packages/apputils-extension/schema/notification.json

```diff
@@ -25,11 +25,23 @@
   },
   "additionalProperties": false,
   "properties": {
+    "checkForUpdates": {
+      "title": "Check for JupyterLab updates",
+      "description": "Whether to check for newer version of JupyterLab or not. If `true`, it will make a request to a website.",
+      "type": "boolean",
+      "default": true
+    },
     "doNotDisturbMode": {
       "title": "Silence all notifications",
       "description": "If `true`, no toast notifications will be automatically displayed.",
       "type": "boolean",
       "default": false
+    },
+    "fetchNews": {
+      "title": "Fetch news from JupyterLab Council",
```

Comment:
How about "Fetch official Jupyter news"? I find the word "Council" more technical than necessary.